Api/schema_v.py

```diff
@@ -5,31 +5,30 @@ def validate_name(data):
     name_re = re.fullmatch(re.compile(r"^\w+$"),data)
     if not name_re:
         raise ValidationError("Visisble charecters only", status_code=400)
-    elif len(data) < 2:
+    elif len(data) <= 2:
         raise ValidationError("name too short")
     
 
-def validate_password(data):
-    if len(data) < 6:
+def validate_password(password):
```

Comment:
Having a function to validate password is fine but it would have been better if you had a regex that checks for cases and lengths of the password. This would have saved you few lines of code which have been taken by the multiple condition statements.